platform/commonUI/general/src/ui/TreeNodeView.js

```diff
@@ -1,155 +1,164 @@
+         * Open MCT, Copyright (c) 2014-2017, United States Government
+         * as represented by the Administrator of the National Aeronautics and Space
+         * Administration. All rights reserved.
+         *
+         * Open MCT is licensed under the Apache License, Version 2.0 (the
+         * "License"); you may not use this file except in compliance with the License.
+         * You may obtain a copy of the License at
+         * http://www.apache.org/licenses/LICENSE-2.0.
+         *
+         * Unless required by applicable law or agreed to in writing, software
+         * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+         * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+         * License for the specific language governing permissions and limitations
+         * under the License.
+         *
+         * Open MCT includes source code licensed under additional open source
+         * licenses. See the Open Source Licenses file (LICENSES.md) included with
+         * this source code distribution or the Licensing information page available
+         * at runtime from the About dialog for additional information.
+         *****************************************************************************/
+
+        define([
+            'zepto',
+            'text!../../res/templates/tree/node.html',
+            './ToggleView',
+            './TreeLabelView'
+        ], function ($, nodeTemplate, ToggleView, TreeLabelView) {
+
+            function TreeNodeView(gestureService, subtreeFactory, selectFn) {
+                this.li = $('<li>');
+
+                this.statusClasses = [];
+
+                this.toggleView = new ToggleView(false);
+                this.toggleView.observe(function (state) {
+                    if (state) {
+                        if (!this.subtreeView) {
+                            this.subtreeView = subtreeFactory();
+                            this.subtreeView.model(this.activeObject);
+                            this.li.find('.tree-item-subtree').eq(0)
+                                .append($(this.subtreeView.elements()));
+                        }
+                        $(this.subtreeView.elements()).removeClass('hidden');
+                    } else if (this.subtreeView) {
+                        $(this.subtreeView.elements()).addClass('hidden');
+                    }
+                }.bind(this));
+
+                this.labelView = new TreeLabelView(gestureService);
+
+                $(this.labelView.elements()).on('click', function (event) {
+                    selectFn(this.activeObject, event);
+                }.bind(this));
+
+                this.li.append($(nodeTemplate));
+                this.li.find('span').eq(0)
+                    .append($(this.toggleView.elements()))
+                    .append($(this.labelView.elements()));
+
+                this.model(undefined);
```

Comment:
Leaving with no arguments will default to undefined

Reply:
Hi @ev1stensberg  This was already there. I have not changed that piece of code.
It is showing green just because spaces are changed.

Reply:
Hi @aayusharora, there seems to be a lot more changes than required. The spacing on the left is not needed.